phpunit.xml.dist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         syntaxCheck="true">
+    <testsuites>
+        <testsuite name="unit">
+            <directory suffix="Test.php">./Tests</directory>
```

Comment:
>   * |tests/| directory, not |Tests|

that would be a strange folder. symfony bundles use PSR-2 and have the
namespace directly in the root folder. this would mean that either we
have a different namespace root nested inside the namespace, or that
tests would have a lower case part in their namespace.

i would opt to keep this one Tests.